pkg/interactive/validation_test.go

```diff
@@ -26,7 +26,7 @@ var _ = Describe("Validation", func() {
 			validator := MinValue(50)
 			err := validator(45)
 			Expect(err).To(HaveOccurred())
-			Expect(err.Error()).To(ContainSubstring("can only validate strings, got int"))
+			Expect(err.Error()).To(ContainSubstring("can only validate strings, got 45"))
```

Comment:
Is this expected to be part of this PR?

Reply:
the tests were never run, was missing the suite runner. Just updated to returned value

Reply:
oh